.drone.star

```diff
@@ -74,29 +74,83 @@ config = {
             "suites": [
                 "apiArchiver",
                 "apiContract",
-                "apiGraph",
-                "apiGraphUserGroup",
-                "apiSpaces",
-                "apiSpacesShares",
                 "apiCors",
                 "apiAsyncUpload",
                 "apiDownloads",
-                "apiReshare",
-                "apiSpacesDavOperation",
                 "apiDepthInfinity",
                 "apiLocks",
+                "apiActivities",
+            ],
+            "skip": False,
+        },
+        "apiGraph": {
+            "suites": [
+                "apiGraph",
+            ],
+            "skip": False,
+            "withRemotePhp": [False],
```

Comment:
I think this will override the default behavior. Should we set it here for this suite or use the default (True for PRs and True,False for cron)?

Reply:
With the defaults, every suite will run for both configs. This is to override the defaults and make the suites run only once, as there are some suites for which we don't need to run the test for both config.